spec/views/social_networking/profile_pages/show.html.erb_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "spec_helper"
+
+module SocialNetworking
+  RSpec.describe "social_networking/profile_pages/show.html.erb", type: :view do
+    fixtures :all
```

Comment:
Based on observations testing these stacks, I believe unit tests should not use fixtures. They should use doubles where possible, or create test data in the test file where necessary. Feature specs probably need fixtures (or possibly factories) because of their complexity, but I think we need to limit the number and scope of feature tests.